Resources/Prototypes/Entities/Structures/Piping/Atmospherics/unary.yml

```diff
@@ -158,7 +158,7 @@
   parent: BaseMachinePowered
   id: BaseGasThermoMachine
   name: thermomachine
-  abstract: true
+  noSpawn: true
```

Comment:
Some of these should actually be abstract. This and gas pipe base and maybe others aren't intended to ever be spawned in and only exist for inheritance

Reply:
noSpawn should only be for things that are actually actively spawned by things in-game, but just shouldn't appear in the spawn menu. E.g., bullets, ghosts, visual-effect-entities

abstract should be things that exist purely for inheritance or something like that, and are never spawned in the game at all.

Reply:
AFAIK the name inheritance issue should be fixed by space-wizards/RobustToolbox/pull/2685, but I havent checked

Reply:
> AFAIK the name inheritance issue should be fixed by [space-wizards/RobustToolbox/pull/2685](https://github.com/space-wizards/RobustToolbox/pull/2685), but I havent checked

Eh, maybe I'll just close this. I asked in discord on what the intended use was because it seems the real issue is probably that the entities I've fixed shouldn't be inheriting their descriptions and names I guess.